lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -1161,6 +1161,9 @@ def parse_element(reply, node)
             parent += '_' + node.parent.attributes['id'] if node.parent.attributes['id']
             parent += '_'
           end
+          if node.name == 'reconciliationID' && reply[:reconciliationID]
+            reply[:reconciliationID2] = node.text unless reply[:reconciliationID] == node.text
```

Comment:
This `unless` clause is probably a smart guard to include; have you seen any transactions that suggest the same ID would be included twice?

Reply:
Hey @jcreiff yes I did notice some transactions did return the same reconciliationID twice in the authReply and captureReply. I thought it'd be a good idea to not double up if that were the case!

Reply:
Interesting! Did you find those transactions "in the wild" or were they in sandbox/testing?

Thinking about it some more, I wonder if we actually want to surface both even if they are the same. If the expectation of an application consuming the response is that a purchase transaction returns two values for auth and capture reply, then the absence of the second ID in the case where they are identical will "feel like" a bug (much like what we have now 😄).

Whereas if the duplicate ID is returned in two places, that's just more information that indicates that both IDs were in fact returned, but they were the same. What do you think?

Reply:
Thanks @jcreiff yep, just by quickly checking out some transaction I did notice a handful of them had duplicates. I was thinking of this as well but I erred on the less noisy side of things. 

I was thinking that reconciliationID_2 will 9 times out of 10 probably be a duplicate given the small sample size of transactions I peeked at. No one's made a fuss about it not being returned prior so I think just surfacing it only when we have the off chance case of it not being a duplicate would be more useful to others. That's my reasoning behind it, but I totally get the 'bug' example you mentioned. I'm totally okay with implementing it to surface all the second ID every time if you feel strongly about it though, let me know!